File: Linear_search/Perl/linear_search.pl
#!/usr/bin/perl
use strict;
use warnings;

my @arr = (1154, 63, 9781, 42, 909);
my $target = 42;
my $index = -1;

for my $i (0 .. $#arr) {
    if ($arr[$i] == $target) {
        $index = $i;
        last;
    }
}
 if ($index != -1) {
     print "The element $target was found at position: $index\n";
 } else {
     print "The element $target was not found\n";
 }

File: Linear_search/Scala/linear_search.scala
object Main extends App {
    var arr = Array(142, 12840, 4, 12, 75433)
    val target = 4
    val index = arr.indexOf(target)

    if (index != -1) {
        println(s"The element $target was found at position ${index + 1}")
    } else {
        println(s"The element $target was not found")
    }
}
